src/Metadata/Property/PropertyMetadata.php

```diff
@@ -30,12 +30,13 @@
     private $writableLink;
     private $required;
     private $iri;
+    private $vocabType;
     private $identifier;
     private $childInherited;
     private $attributes;
     private $subresource;
 
-    public function __construct(Type $type = null, string $description = null, bool $readable = null, bool $writable = null, bool $readableLink = null, bool $writableLink = null, bool $required = null, bool $identifier = null, string $iri = null, $childInherited = null, array $attributes = null, SubresourceMetadata $subresource = null)
+    public function __construct(Type $type = null, string $description = null, bool $readable = null, bool $writable = null, bool $readableLink = null, bool $writableLink = null, bool $required = null, bool $identifier = null, string $iri = null, string $vocabType = null, $childInherited = null, array $attributes = null, SubresourceMetadata $subresource = null)
```

Comment:
Changing the argument order is a backward compatible break that we can't add without a new major version release. One of way adding arguments without BC-break is to add it at the end of the constructor, with a default `null` value.